content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,403 @@
+### Introduction
+
+---
+
+A Coming Soon landing page is a temporary home page that informs visitors that your website is in the process of being built or will be available soon.
+Since a coming soon page lacks a header or a footer, visitors cannot navigate to the rest of your website hence it is important to capture what is under development so as to give as much information as possible.
+Every coming soon page includes a statement encouraging visitors to return at a later time, as well as any other pertinent information concerning the website that is under development.
+This article will teach you how to make a coming soon landing page but first things first you will learn about Javascript Timers to get you started.
+
+#### Table Of Contents
+
+---
+
+- [introduction](#introduction)
+- [Javascript Timers overview](#javascript-timers-overview)
+- [code execution after a delay](#code-execution-after-a-delay)
+- [Code Execution at Regular Intervals](#code-execution-at-regular-intervals)
+- [Cancelling A Timer](#cancelling-a-timer)
+- [Template section](#template-section)
+- [Best practices](#best-practices)
+- [conclusion](#conclusion)
+
+### Javascript Timers Overview
+
+---
+
+A timer is a feature that allows us to run a program at a predetermined time.
+Timers can be used to postpone code execution so that it doesn't finish at the same time as an event or a page loading.
+Timers, for example, can be used to alter your website's advertisement banners at regular intervals or to display a real-time clock, among other things. In JavaScript, there are two timer functions: `setTimeout ()` and `setInterval ()`. Browsers implement timer functions, and their implementations differ from one browser to another. The Node.js runtime also has a native implementation of timers. In browsers, the window interface is the parent of the main timer features. Because the window interface makes its elements available globally in the main javascript scope, setTimeout may be used in the browser console.
+'Timers' is an object in NodeJs that functions like the browser's window interface.
+
+### Code Execution After a Delay

```

Comment:
Heading capitalization : i.e in this case it should be : `Code execution after a delay`. Make changes where applicable thoughout your article.